data/json/monster_factions.json

```diff
@@ -307,7 +311,8 @@
     "type": "MONSTER_FACTION",
     "name": "spider",
     "base_faction": "insect",
-    "by_mood": "spider"
```

Comment:
This makes spiders default to treating each other as "friendly". Is this intended, and if so, why?

Reply:
Damn, thanks! I was going to "while I'm here" a lot of monfaction stuff in preparation for the other reworks before realizing that that's kinda bad practice. I forgot to return them to the status quo.

Reply:
Understand fully (sounds like something I might do...); quite welcome!